### DISCUSSION TITLE
defer the filters selects and table search

### DISCUSSION THREAD

**Discussion by ariaieboy** [filamentphp/filament#6682] [Ideas]
With each change in the filters or search fields, the table content gets refreshed.
Can all filters and search use defer mode, and only the filters apply when the user presses a button?

**Comment by fodor11** [comment]
Invoke `->searchOnBlur(true)` method on the `$table` to make searches only fire on hitting enter
Invoke `->deferFilters()` on the `$table` to apply filters only on clicking the Apply button (see the docs: https://filamentphp.com/docs/3.x/tables/filters/getting-started#deferring-filters)
